tests/trilinos/trilinos_sparsity_pattern_03.cc

```diff
@@ -39,7 +39,7 @@ void test ()
   columns.add_range(3*myid, 3*myid+3);
   columns.compress();
 
-  sp.reinit(rows, columns, MPI_COMM_WORLD, 0);
+  sp.reinit(rows, columns, MPI_COMM_WORLD, (dealii::types::global_dof_index)0);
```

Comment:
http://cdash.kyomu.43-1.org/testDetails.php?test=20521173&build=11899

It basically tries to use the ```(IndexSet, IndexSet, CSP, MPI_COMM, bool)``` template function with default bool argument. This is on clang and intel!